fvm/src/syscalls/mod.rs

```diff
@@ -329,14 +329,22 @@ where
     }
 }
 
-impl<C> SyscallHandler<DefaultFilecoinKernel<C>> for DefaultFilecoinKernel<C>
+impl<K> SyscallHandler<K> for DefaultFilecoinKernel<K::CallManager>
 where
-    C: CallManager,
+    K: FilecoinKernel
+        + ActorOps
+        + IpldBlockOps
+        + CryptoOps
+        + DebugOps
+        + EventOps
+        + GasOps
+        + MessageOps
+        + NetworkOps
+        + RandomnessOps
+        + SelfOps,
```

Comment:
This made no sense until I noticed  https://github.com/filecoin-project/ref-fvm/pull/1941 :smile: